components/commercial_filter/views/popular_artists/index.styl

```diff
@@ -22,7 +22,7 @@
       white-space nowrap
       overflow hidden
       text-overflow ellipsis
-      max-width 175px
+      width calc(66% \- 15px)
```

Comment:
Good ol' calc!

Random question... is this dividing by `-15px` or is `\-` some operator I don't know about?